docs/book/src/cronjob-tutorial/pprof-tutorial.md

```diff
@@ -0,0 +1,61 @@
+# Using Pprof for Controller Profiling
+
+Pprof is a Go based library designed for profiling the binary in order to gather CPU, and Memory related data. Pprof is helpful in finding any sort of bottlenecks that are appearing either at Memory, CPU, or both levels.
+
+Pprof itself runs as part of the the Controller. You can extract the statistics, and visualize them on an interactive website. Furthermore, **you don't need to install this package seperately**. Pprof comes packaged as part of Kubebuilder `controller-runtime` [dependency](https://github.com/kubernetes-sigs/controller-runtime/pull/1943) within the scafolded project.
+
+For further information about the Pprof usage, check out the official [docs](https://github.com/google/pprof).
+
+# How to use Pprof in your Codebase
+
+1. In your `cmd/main.go` file, specify the `PprofBindAddress` URL field as part of Controller Manager's configurations:
+
+    ```golang
+    mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+      ....
+      Scheme:                 scheme,
+      LeaderElection:         enableLeaderElection,
+      // PprofBindAddress is the TCP address that the controller should bind to
+      // for serving pprof. Specify the manager address and the port that should be bind.
+      PprofBindAddress:       "127.0.0.1:8082",
```

Comment:
```suggestion
      PprofBindAddress:       ":8082", // Set the port where pprof will be available
```

I think you do not need set the localhost IP just the port to bind

Reply:
@camilamacedo86 `:8082` means that it listens on `localhost` or `0.0.0.0`? IMO, keeping the IP addresses would be much helpful. It's standard syntax used across all apps that involve HTTP server setup.

Reply:
In short, use `0.0.0.0` instead of `127.0.0.1`?

Reply:
In short, if we use `:8082` that works for both as fair I am aware of.

Reply:
Gonna verify this first before pushing any changes.

Reply:
@camilamacedo86  `:8082` works just fine. I've updated the document.